src/components/NewCommentForm/NewCommentForm.tsx

```diff
@@ -1,39 +1,84 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './NewCommentForm.scss';
+import { nanoid } from 'nanoid';
+import { Comment } from '../../types/comment';
 
-export const NewCommentForm: React.FC = () => (
-  <form className="NewCommentForm">
-    <div className="form-field">
-      <input
-        type="text"
-        name="name"
-        placeholder="Your name"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <div className="form-field">
-      <input
-        type="text"
-        name="email"
-        placeholder="Your email"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <div className="form-field">
-      <textarea
-        name="body"
-        placeholder="Type comment here"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <button
-      type="submit"
-      className="NewCommentForm__submit-button button"
-    >
-      Add a comment
-    </button>
-  </form>
-);
+type Props = {
+  postId: number;
+  addNewComment: (newComment: Comment) => void;
+};
+
+export const NewCommentForm: React.FC<Props> = ({ postId, addNewComment }) => {
+  const [newName, setNewName] = useState<string>('');
+  const [newEmail, setNewEmail] = useState<string>('');
+  const [newBody, setNewBody] = useState<string>('');
```

Comment:
```suggestion
  const [newName, setNewName] = useState<>('');
  const [newEmail, setNewEmail] = useState<>('');
  const [newBody, setNewBody] = useState<>('');
```

You don't need to be explicit here)